packages/@o3r/testing/src/core/angular/angular-materials/slider-material.ts

```diff
@@ -0,0 +1,19 @@
+import { DebugElement } from '@angular/core';
+import { MatSliderProfile } from '../../angular-materials';
+import { O3rElement } from '../element';
+import { O3rSliderElement } from '../elements';
+
+export { SelectElementProfile } from '../../elements';
+
+/**
+ * Implementation dedicated to angular / TestBed.
+ */
+export class MatSlider extends O3rSliderElement implements MatSliderProfile {
+  constructor(sourceElement: DebugElement | O3rElement) {
+    super(
+      sourceElement instanceof O3rElement ? sourceElement.sourceElement : sourceElement,
+      '.mdc-slider__track',
```

Comment:
Could be use from `cssClasses.TRACK` (from `import { cssClasses } from '@material/slider';`)